### ISSUE TITLE
meta-png v1.0.1 module issue?

### ISSUE THREAD

**Issue by phonyexpress** [lucach/meta-png#2]
I see the new meta-png v1.0.1 on https://www.npmjs.com/package/meta-png.  When I do an npm install meta-png@1.0.1, I get only the package.json and the README.md.  However, when I npm install meta-png@1.0.0, I get the package.json and the README.md, but also a 'dist' directory with the javascript files in it.

Is there an issue with the meta-png v1.0.1 module, at least the one on npmjs.com?  Of course, I could have something messed up in my environment, but I reviewed my setup and I'm not seeing this issue with other modules, so I though it best to ask if anybody else is seeing this issue.

Thank you.

**Comment by lucach** [comment]
Hi @phonyexpress ,
thanks for noticing this! It was an embarrassing error on my side: I released 1.0.1 a couple of days ago after a PR that added typescript information without rebuilding.

I've now released and published version 1.0.2, which should be non-empty. And [with this commit I've also added a hook](https://github.com/lucach/meta-png/commit/3ea415b8c5cd82c6e23c26a6401d788caeb0f33c) to automatically build before releasing, to prevent this kind of mistake in the future.

Sorry for the inconvenience and thanks again for pointing it out!
